features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP.cpp

```diff
@@ -62,6 +66,10 @@ static const intptr_t cellular_properties[AT_CellularDevice::PROPERTY_MAX] = {
     0,  // PROPERTY_NON_IP_PDP_TYPE
     1,  // PROPERTY_AT_CGEREP
     1,  // PROPERTY_AT_COPS_FALLBACK_AUTO
+    0,  // PROPERTY_SOCKET_COUNT
```

Comment:
Same like mentioned above , `PROPERTY_SOCKET_COUNT` is 7, `PROPERTY_IP_TCP` and `PROPERTY_IP_UDP` is 1.

Reply:
Please see my comment above.